### ISSUE TITLE
Supports Window Object, Console Object and several W3C standards

### ISSUE THREAD

**Issue by angelozerr** [angelozerr/tern-tabris#24]
According http://eclipsesource.com/blogs/2015/05/15/javascript-api-top-tabris-js-feature-5/ tabris supports windows, object, etc

@cpetrov is window can be accessible without tabris object? If it that, it will be easy to do that by referencing tabris.window to window of browser.json.

**Comment by cpetrov** [comment]
Yes, but it doesn't support all functions and properties provided by the `browser.json` definition.
